### PR TITLE
Fix lots of parameters

### PR DESCRIPTION
**Pull request by jg75**
Devops PR: https://github.com/AmberEngine/docker/pull/77
Confetti PR: https://github.com/AmberEngine/confectionary-confetti/pull/7
Morpheus PR: https://github.com/AmberEngine/morpheus/pull/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +34,7 @@ The default value will be the class name in either case.
     config = Confetti(session=session)
 ```
 
-## Or you could just extend Confetti in your application
+## Example: extend Confetti in your application
```
Comment (clagraff) [inline README.md]: might wanna include some text afterwords about _why_ someone may want to extend Confetti. 'Cause honestly im not sure I understand why I'd need to?
Do I _have_ to? Or is there simply some circumstances where I would potentially want to?

File: README.md [hunk 2]
```diff
@@ -81,16 +79,26 @@ see: http://boto3.readthedocs.io/en/latest/reference/services/ssm.html#SSM.Clien
...
 
-    # Reload from AWS SSM
-    config.get_parameters()
+## Or export parameters to a JSON file so you can modify in bulk.
+```python
+    # Create YourApp's config
+    config = YourApp()
+
+    # Export to JSON
+    config.export_parameters('example.json')
```
Comment (clagraff) [inline README.md]: Can I use `config` to save new parameters individually? Or is my only option to use `import_parameters`?

Comment (clagraff) [inline README.md]: You show us how to import and export parameters.
Can I remove/delete existing parameters? If so, how?

File: confetti/confetti.py [hunk 3]
```diff
@@ -100,10 +101,12 @@ def get_parameters(self):
         """Get parameters from AWS SSM parameter store by path."""
         self.parameters = dict()
         ssm = self.session.client('ssm')
+        parameters = ssm_utils.get_parameters_by_path(
+            ssm,
+            **{'Path': self.confetti_path}
```
Comment (clagraff) [inline confetti/confetti.py]: why not just do `Path=self.confetti_path` instead of exploding out the dictionary?

File: confetti/confetti.py [hunk 4]
```diff
@@ -192,6 +195,26 @@ def import_parameters(self, file_name):
         if parameters:
             self.put_parameters(parameters)
 
+    def export_parameters(self, file_name):
+        """Import parameters to a JSON file.
```
Comment (clagraff) [inline confetti/confetti.py]: Not a huge deal, but that seems like a funny way of phrasing your docstring.

For _export_ we do an _import_. 🤣 
Maybe rephrase like `Write available parameters to a JSON file.` instead?

File: README.md (hunk 1 above)
Comment (jg75) [reply README.md]: I could put a more robust example of a docker service provisioner.

File: README.md (hunk 2 above)
Comment (jg75) [reply README.md]: it's your option.  it's a one time thing and you don't have to use confetti at all if you don't want to.  you could do it through the aws console in systems manager -> parameter store

Comment (jg75) [reply README.md]: currently only through the console, but we could add more functionality to confetti if we need it.  It's not needed for provisioning, because provisioning will always be starting from 0 and importing N parameters everytime.  If I need to delete them it probably won't be programmatically.

File: confetti/confetti.py (hunk 3 above)
Comment (jg75) [reply confetti/confetti.py]: hmm yeah, not sure what I was thinking there.

File: confetti/confetti.py (hunk 4 above)
Comment (jg75) [reply confetti/confetti.py]: Oops.

File: README.md [hunk 5]
```diff
@@ -81,16 +79,26 @@ see: http://boto3.readthedocs.io/en/latest/reference/services/ssm.html#SSM.Clien
 
     # Import from JSON
     config.import_parameters('example.json')
```
Comment (dougnorton) [inline README.md]: Is it intended behavior that when I use `import_parameters` if the parameter already exists I get a `ParameterAlreadyExists` error. Maybe there should be an kwarg to optionally overwrite the value?

File: README.md [hunk 6]
```diff
@@ -81,16 +79,26 @@ see: http://boto3.readthedocs.io/en/latest/reference/services/ssm.html#SSM.Clien
...
+```
```
Comment (dougnorton) [inline README.md]: Is it intended behavior that when I use import_parameters if the parameter already exists I get a ParameterAlreadyExists error. Maybe there should be an kwarg to optionally overwrite the value?

Comment (jg75) [reply README.md]: Overwrite=True

File: README.md (hunk 2 above)
Comment (clagraff) [reply README.md]: maybe throw in a disclaimer that saving individual parameters and stuff can be done through the AWS console. Maybe include a link since it is hard to get to 👍

Comment (clagraff) [reply README.md]: see above comment

> maybe throw in a disclaimer that saving individual parameters and stuff can be done through the AWS console. Maybe include a link since it is hard to get to
